apps/site/pages/docs/component-customization/creating-component.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Creating a Component'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Creating a Component
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+In this guide, you'll learn how to create a component from scratch and use it in your FastStore project. For the purpose of this guide, we'll create a `Price` component to display the price of a product in the `Product Details` component.
+
+|   |   |
+|---|---|
+| ![faststore-component](https://vtexhelp.vtexassets.com/assets/docs/src/faststore-price-component___d9d0e2fbf65e6680d0f26bed7a0e9b8e.png) The **FastStore `Price` component** displays the price of a given product, discount and total values.| ![custom-component](https://vtexhelp.vtexassets.com/assets/docs/src/custom-price-component___3b8a888867b6bf354541236163ef3d79.png) The **custom `Price` component** created can display any information that you set. In this case it renders the name `Price`|
+
+## Step by step
+
+### Step 1: Create the `Price` file
+
+1. Open your FastStore project in a code editor and navigate to `src` folder.
+2. Create a new folder called `components` inside the `src` folder.
+3. Inside the `components` folder, create a new file called `Price.tsx`.
+4. Export the following function in the `Price.tsx` file:
+
+```tsx
+
+export default function Price(props: any) {
+  return <div>Price</div>;
+}
+
+```
+
+<Callout type="info" emoji="ℹ️">
+  This function creates the new `Price` component. When this component is used, it displays the word `Price` on the screen. You can change what the component displays by editing the information within the `<div>` tags. 
+</Callout>
+
+To use the new `Price` component, you need to override the default `Price` component by following the next step.
+
+## Step 2: Override the `Price` component
+Now, let's override the component to render it in the Product Details.
+
+1. Inside the `components` folder, create a new folder called `overrides`.
+2. In the `overrides` folder, create a new file called `ProductDetails.tsx`.
+3. Export the following function in the `ProductDetails.tsx` file to override the `Price` component:
+
+```tsx
+import CustomPrice from '../Price'
```

Comment:
How about we change this filename to `CustomPrice` as well?

Reply:
My fear here is that there will be an assumption that the name of the file of the component we're creating has to match the name of the component that component will override.

Reply:
Good point, @gvc! I'm taking note of this so we can add it to a (future) Best Practice doc for Overriding Components as well.